charts/argo-workflows/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: v3.4.9
+appVersion: v3.4.10
```

Comment:
3.4.10 is not released yet: https://github.com/argoproj/argo-workflows/issues/11552.
Will probably be out after Tuesday's contributor meeting

Reply:
https://github.com/argoproj/argo-helm/pull/2220